### PR TITLE
Add entity_update command for Drupal 8

### PR DESCRIPTION
**Pull request by mgalang**

### CODE REVIEW

**Comment by badrange** [comment]
Would it make sense to add it to commands.yml too so that it gets run automatically upon build.sh update? This looks like one thing that would easily be forgotten (just like running drush updb which now happens automatically..)

File: build.sh [hunk 1]
```diff
@@ -302,6 +302,17 @@ class Maker:
 		else:
 			self.warning("Unable to update")
 
+	# Run entity updates. This should be run after update command.
+	def entity_update(self):
+		if self._drush([
+			"--root=" + format(self.final_build_dir + self.drupal_subpath),
```
Comment (tharna) [inline build.sh]: Root will be moved to _drush function, so this should be removed. See #35

**Comment by tharna** [comment]
Example of this could be in commands.yml, but need to note this is only for d8.

**Comment by mgalang** [comment]
@badrange actually this should be part of the entity_update command. It needs to be executed after drush updb for D8. So optionally it should be in the commands.yml

@tharna I see. I'll update this branch again soon

**Comment by badrange** [comment]
If you add it to `commands.yml`'s update command (which was what I meant above) I suggest also putting it to WunderTools' d8 version of `commands.yml`so that it will be in use for all new projects.. :)

**Comment by mgalang** [comment]
@badrange Sure, I'll add it to wundertools d8 when it's merged.

**Comment by mgalang** [comment]
Closing this for now. Let's just add it optionally in commands yml via drush command

**Pull request closed without merging**